src/ctapipe/image/extractor.py

```diff
@@ -1300,8 +1300,7 @@ class VarianceExtractor(ImageExtractor):
 
     """
     Extractor that calculates
-    the variance of each waveform and
-    corrects for the relative gain.
+    the variance of each waveform
```

Comment:
This has to be a _single_ line, really concise. A longer description can follow after one blank line.

Reply:
sorry about that. i shortened it.